src/modules/previewpane/common/Properties/AssemblyInfo.cs

```diff
@@ -36,5 +36,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("1.0.0.1")]

```

Comment:
nit: Why updating the assembly version

Reply:
Because there was an issue installing it on my system. But I will update it to default for consistency purpose.